package.json

```diff
@@ -19,38 +19,32 @@
     "lint": "npx tslint -c tslint.json --project . && npx prettier --list-different './packages/**/src/*.ts'"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.4",
-    "@babel/preset-env": "^7.7.6",
-    "@babel/runtime-corejs3": "^7.7.6",
-    "@polkadot/dev": "^0.31.1",
-    "@polkadot/ts": "^0.1.86",
-    "@types/jest": "^23.3.10",
-    "babel-jest": "^24.9.0",
+    "@babel/core": "^7.10.5",
+    "@babel/preset-env": "^7.10.4",
+    "@babel/runtime-corejs3": "^7.10.5",
+    "@polkadot/dev": "^0.55.27",
+    "@polkadot/ts": "^0.3.29",
+    "@types/jest": "^26.0.7",
+    "babel-jest": "^26.1.0",
     "core-js": "^3.4.8",
-    "cz-lerna-changelog": "^2.0.2",
+    "cz-lerna-changelog": "^2.0.3",
     "husky": "^1.1.1",
-    "jest": "24.1.0",
-    "jest-environment-node": "^24.0.0",
-    "lerna": "^3.13.1",
+    "jest": "^26.1.0",
+    "jest-environment-node": "^26.1.0",
+    "lerna": "^3.22.1",
     "prettier": "^1.13.5",
     "pretty-quick": "^1.11.1",
-    "ts-jest": "^24.0.0",
-    "ts-node": "^8.0.3",
-    "tsconfig-paths": "^3.8.0",
-    "tslint": "^5.14.0",
-    "tslint-microsoft-contrib": "^5.2.1",
-    "typedoc": "^0.14.2",
-    "typedoc-plugin-monorepo": "https://github.com/ianhe8x/typedoc-plugin-monorepo"
-  },
-  "resolutions": {
-    "babel-core": "^7.0.0-bridge.0",
-    "typescript": "^3.7.2",
-    "braces": "^2.3.2",
-    "fstream": "^1.0.12",
-    "handlebars": "^4.5.3",
-    "js-yaml": "^3.13.1",
-    "marked": "^0.6.2",
-    "tar": "^4.4.8"
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-is": "^16.13.1",
+    "semantic-release": "^15.8.0",
+    "styled-components": "^5.1.1",
+    "ts-jest": "^26.1.3",
+    "ts-node": "^8.10.2",
+    "tsconfig-paths": "^3.9.0",
+    "typedoc": "^0.17.8",
+    "typedoc-plugin-monorepo": "^0.1.0",
+    "typescript": "^3.9.7"
```

Comment:
I suppose the main purpose of this pull request is to particularly upgrade Pokadot dependencies, so that cennznet/api.js is able to be synchronised with  latest Pokadot dependencies.

IMO, to get the job be easier validated and tracked, it may be worth to try upgrade Polkadot related dependencies only in this PR; After that works, we could create a separated ticket/PR to upgrade non Polkadot related dependencies. On the other side, it could get reviewing job much easier and the PR could be done more effectively. What do you think?

Reply:
As what we discussed in stand-up meeting, ignore this comment if it is already working in this PR. 

Otherwise, would reckon taking gradually upgrading packages in future PRs.